woodwork/tests/logical_types/test_logical_types.py

```diff
@@ -38,11 +38,19 @@ def test_instantiated_type_str():
 
 
 def test_ordinal_order_errors():
+    series = pd.Series([1, 2, 3]).astype('category')
+    error = "Must use an Ordinal instance with order values defined"
+    with pytest.raises(TypeError, match=error):
+        series.ww.init(logical_type='Ordinal')
+
+    with pytest.raises(TypeError, match=error):
+        series.ww.init(logical_type=Ordinal)
+
     with pytest.raises(TypeError, match='Order values must be specified in a list or tuple'):
-        Ordinal(order='not_valid')
+        series.ww.init(logical_type=Ordinal(order='not_valid'))
```

Comment:
might be more related to the logical type tests to do `Ordinal(order='not_valid').transform(series)` here and in the other check below.

Though the top two checks here can't be done outside of the series init... but I think they get checked in `test_column_accessor` so I'm not sure they're necessary

Reply:
Yeah, makes sense. I removed the two that are tested in `test_column_accessor` and updated the others to use `transform` as you suggested.